src/installer/pkg/packaging/osx/sharedframework/shared-framework-distribution-template.xml

```diff
@@ -7,7 +7,7 @@
     <conclusion file="conclusion.html" mime-type="text/html" />
     <volume-check>
         <allowed-os-versions>
-            <os-version min="10.12" />
+            <os-version min="10.13" />
```

Comment:
Do you know how this interacts with the Core-SDK os-version min?

https://github.com/dotnet/core-sdk/blob/0e12ef55274c3533f49a90ec931dedc9bec39fed/src/redist/targets/packaging/osx/clisdk/Distribution-Template#L10

```xml
    <volume-check>
        <allowed-os-versions>
            <os-version min="10.12" />
        </allowed-os-versions>
    </volume-check>
```

Reply:
Will try to build the SDK and see what happens - most likely the runtime will be skipped but SDK installation would succeed.

Reply:
Created an SDK issue to fix the minimum OSX version: https://github.com/dotnet/core-sdk/issues/6742